src/index.ts

```diff
@@ -57,6 +61,10 @@ export let AppState: AppState = {
   loaderAnimationEndTimeout: undefined,
   lastLoadedVideoId: null,
   lyricAbortController: null,
+  animationSettings: {
+    disableRichSynchronization: false,
+    lineSyncedWordDelayMs: 0.05,
```

Comment:
i think the default used to be 50ms


```suggestion
    lineSyncedWordDelayMs: 50,
```